zeebe/qa/util/src/main/java/io/camunda/zeebe/qa/util/cluster/TestCamundaApplication.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.qa.util.cluster;
+
+import io.camunda.application.configuration.ApplicationModuleConfiguration;
+import io.camunda.commons.CommonsModuleConfiguration;
+import java.util.List;
+
+public abstract class TestCamundaApplication<T extends TestCamundaApplication<T>>
```

Comment:
💭 Maybe you can review https://github.com/camunda/camunda/pull/18964 then we can change it on the new camunda app 🤔

Reply:
Will do.